app/mailers/notification_mailer.rb

```diff
@@ -0,0 +1,15 @@
+class NotificationMailer < ApplicationMailer
+  def send_notifications(user, 
+					  							task, 
+					  							task_category, 
+					  							notification_taskee, 
+					  							notification_tasker
```

Comment:
Align the parameters of a method definition if they span more than one line.<br>Tab detected.